proto-lens-protoc/app/Data/ProtoLens/Compiler/Editions/Features.hs

```diff
@@ -48,7 +48,7 @@ for a particular edition.
 featuresForEditionFromDefaults :: FeatureSetDefaults -> Edition -> FeatureSet
 featuresForEditionFromDefaults defaults edition
   | (d : _) <- candidates = (d ^. #overridableFeatures) `mergedInto` (d ^. #fixedFeatures)
-  | otherwise = defMessage
+  | otherwise = error $ "Unsupported edition with tag number: " ++ show (fromEnum edition)
```

Comment:
I would rather not use `error`. Can you work some better error handling in here?

Reply:
Changed this to return an `Either Text a`, and the call chain has been updated accordingly all the way up to `makeResponse`, which will now include the error string in the response.